azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTest.java

```diff
@@ -225,14 +234,8 @@ public void exec1FailedKillAll() throws Exception {
     Assert.assertTrue(runner.isKilled());
 
     Assert.assertTrue(
-        "Expected flow " + Status.FAILED + " instead " + exFlow.getStatus(),
-        exFlow.getStatus() == Status.FAILED);
-
-    try {
-        Thread.sleep(500);
```

Comment:
I'll give it a try. First of this test should use some custom job type for tests, instead of type=java jobs that have a lot of overhead in launching.